### PR TITLE
fix: argo logs completion

### PR DESCRIPTION
**Pull request by zhangzhiqiangcs**
Fix arog logs command completion

### CODE REVIEW

**Review by terrytangyuan — COMMENTED** [review]
Can you paste the output of the new completions?

**Comment by zhangzhiqiangcs** [comment]
> Can you paste the output of the new completions?

First tab, will output workflow suggestions. Second tab, will output pod list of the workflow.

image

Total completion content has too many lines. Hers is the changed part
```bash
__argo_get_logs() {
    # Determine if were completing a workflow or not.
    if [[ $prev == "logs" ]]; then     
          __argo_get_workflow && return $?
    fi
    local workflow=$prev  
    # Otherwise, complete the list of pods
    local -a kubectl_out
    if kubectl_out=($(kubectl get pods --no-headers --selector=workflows.argoproj.io/workflow="${workflow}" 2>/dev/null | awk '{print $1}' 2>/dev/null)); then
        COMPREPLY+=( $( compgen -W "${kubectl_out[*]}" -- "$cur" ) )
    fi
}
```

File: cmd/argo/commands/completion.go [hunk 1]
```diff
@@ -42,22 +42,13 @@ __argo_get_cron_workflow() {
...
-
-	# If completing a workflow, call normal function.
-	if [[ $workflow -eq 1 ]]; then
-		__argo_get_workflow && return $?
-	fi
-
+	if [[ $prev == "logs" ]]; then
+          __argo_get_workflow && return $?
+ 	fi
+    local workflow=$prev
```
Comment (terrytangyuan) [inline cmd/argo/commands/completion.go:48]: Wrong indentation here

Comment (zhangzhiqiangcs) [reply cmd/argo/commands/completion.go:48]: fixed

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**